build.sbt

```diff
@@ -2,6 +2,7 @@
 import sbtcrossproject.CrossPlugin.autoImport.crossProject
 import Scalaz._
 import xerial.sbt.Sonatype._
+import explicitdeps.ExplicitDepsPlugin.autoImport.moduleFilterRemoveValue
```

Comment:
No clue why that is but I couldn't even start sbt without this import

Reply:
For me, global coursier causes this

Reply:
I don't have global coursier. But I have global sbt-updates and sbt-license-report. I will try later today if it's any of them. Unfortunate that this could happen.

Reply:
I finally found some spare time to try it out - it's sbt-updates. We could revert this commit, but on the other hand it doesn't hurt either.